src/openrct2/windows/top_toolbar.c

```diff
@@ -361,6 +364,12 @@ static void window_top_toolbar_mouseup(rct_window *w, sint32 widgetIndex)
 	case WIDX_NEWS:
 		window_news_open();
 		break;
+	case WIDX_MUTE:
+		if(gGameSoundsOff)
+			audio_unpause_sounds();
+		else
+			audio_pause_sounds();
+		break;
```

Comment:
I'd rather see this correspond with the Sound checkbox in the Options window. In order to achieve this, please change this if statement as follows:
```
	case WIDX_MUTE:
		gConfigSound.sound_enabled = !gConfigSound.sound_enabled;
		config_save_default();
		break;
```